elements/pl-symbolic-input/pl-symbolic-input.py

```diff
@@ -70,8 +72,12 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     element = lxml.html.fragment_fromstring(element_html)
     name = pl.get_string_attrib(element, "answers-name")
     label = pl.get_string_attrib(element, "label", LABEL_DEFAULT)
-    variables_string = pl.get_string_attrib(element, "variables", VARIABLES_DEFAULT)
-    variables = phs.get_variables_list(variables_string)
+    variables = phs.get_items_list(
+        pl.get_string_attrib(element, "variables", VARIABLES_DEFAULT)
+    )
+    custom_functions = phs.get_items_list(
+        pl.get_string_attrib(element, "custom-functions", CUSTOM_FUNCTIONS_DEFAULT)
+    )
```

Comment:
Yes; the changes here are pretty far-reaching and there could still be changes to how this is all set up based on feedback. I'm holding off on adding docs until we're sure this will be the final behavior. There still needs to be documentation on how the assumptions system is set up and some discussion of what's allowed by the new parser.

Reply:
Docs have been added as of latest commit.